### ISSUE TITLE
Could you share the model trained for subtask2?

### ISSUE THREAD

**Issue by huangmozhilv** [JunMa11/AbdomenCT-1K#20]
Great job and thanks for the kindness to share the code and baseline models! 
We have a dataset consisting of multi-phase contrast enhanced abdominal CT scans, and would like to segment the livers and spleens. The model trained for subtask2, which was trained with data from multi-phases, multi-vendor and multi-centers, should perform well for our dataset. 

The models in BaiduNetDisk were named 'Task311_OrganSparse5',  'Task312_OrganSparse30',  'Task313_OrganSparse15', what are their differences? According to their pkl files, the training files seem to be the 41 MSD spleen cases  ('Task314_SpleenSparse15' ). Thus they don't seem to be the mdoels trained for subtask2.

Could you make the model trained subtask2 public? Thank you very much.

**Comment by JunMa11** [comment]
the models you mentioned were trained with different label ratios. Please look at the paper for more details.

Which subtask2 are you looking for? e.g., which benchmark?

**Comment by huangmozhilv** [comment]
I am looking for the model of subtask 2 for the `Fully supervised abdominal organ segmentation benchmark`. The training set consist of 3 datasets: MSD Pan. Plus (281), LiTS Plus (40), KiTS Plus (40).

**Comment by JunMa11** [comment]
Here you go (T122)
链接：https://pan.baidu.com/s/1BNZn4jkgbRWzIP7XXL6Tcg?pwd=2022 
提取码：2022 

BTW, docker would be a more convenient way to use the trained model. You do not need to set up the environment again.

**Comment by huangmozhilv** [comment]
Thank you very much. I'm not familiar with docker. Will try it later.

**Comment by JunMa11** [comment]
It will save your life (time) to use the dockerized models.

You only need to install docker and nvidia-docker. Then, you can run the inference with one-line command.

Here is a good tutorial on docker
https://nbviewer.org/github/ericspod/ContainersForCollaboration/blob/master/ContainersForCollaboration.ipynb

**Comment by huangmozhilv** [comment]
Thanks so much. You are so nice to share the tutorial.
